content/800-data-platform/050-data-proxy/900-considerations-limitations.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: 'Considerations and limitations'
+metaTitle: 'Considerations'
+metaDescription: 'When you use Data Proxy for applications in serverless and edge runtimes, you should bear in mind a few considerations and learn about the current limitations.'
```

Comment:
```suggestion
metaDescription: 'When you use Data Proxy for applications in serverless and edge runtimes, bear in mind a few considerations and learn about the current limitations.'
```

Shorter, and [avoids should](https://www.prisma.io/docs/about/prisma-docs/style-guide/word-choice#avoid-ambiguous-english-words)